make.ps1

```diff
@@ -177,6 +178,19 @@ elseif ($command -eq "check")
 		UtilityNotFound
 	}
 }
+elseif ($command -eq "check-scripts")
+{
+	foreach ($script in ls "mods/*/maps/*/*.lua")
+	{
+		luac $script
+	}
+	foreach ($script in ls "lua/*.lua")
+	{
+		luac $script
+	}
+	rm luac.out
```

Comment:
This was just to remove the `luac.out` file generated by the check, but the `-p` flag does this better/anyway.